### PR TITLE
Added more logs and clean up src/shibuya.js

### PR DESCRIPTION
**Pull request by chrisli30**

### CODE REVIEW

File: src/common/turingHelper.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import _ from 'lodash';
 import { rpc, types, runtime } from '@oak-network/types';
-import { ApiPromise, WsProvider, Keyring } from '@polkadot/api';
+import { ApiPromise, WsProvider } from '@polkadot/api';
 import { u8aToHex } from '@polkadot/util';
+import Keyring from '@polkadot/keyring';
```
Comment (chrisli30) [inline src/common/turingHelper.js:5]: The Keyring in @polkadot/api is a wrapper. Let’s use the real one underneath it instead.

File: src/common/turingHelper.js [hunk 2]
```diff
@@ -29,38 +31,6 @@ class TuringHelper {
 
     getTokenBalance = async (address, tokenId) => this.api.query.tokens.accounts(address, tokenId);
 
-    // TODO: this function doesn’t look correct. It can be replicated from mangataHelper, calling getProxyAccount in utils.js
-    getProxyAddress = (address, paraId) => {
```
Comment (chrisli30) [inline src/common/turingHelper.js:33]: This is not used and no long needed with turingHelper.getProxyAccount

File: src/shibuya.js [hunk 3]
```diff
@@ -64,7 +66,7 @@ const scheduleTask = async ({
     const xcmpExtrinsic = shibuyaHelper.createTransactExtrinsic({
         targetParaId: TuringDev.paraId,
         encodedCall: encodedTaskViaProxy,
-        proxyAccount: proxyOnTuring,
+        proxyAccount: keyring.decodeAddress(proxyOnTuring),
```
Comment (chrisli30) [inline src/shibuya.js:69]: `keyring.decodeAddress(proxyOnTuring)` returns accountId, which is needed by this call.

The rest of time proxy address needs to be encoded and looks like a Shibuya or Turing address, such as `aNFdmcr5ERyubSoB6mqb2q4qYKT8pPTLnFe8Q2oGeKj7Sze`

File: src/shibuya.js [hunk 4]
```diff
@@ -106,58 +107,79 @@ const main = async () => {
...
     // We also need to transfer tokens to the proxy account to pay for XCM and task execution fees
     console.log('\n1. One-time proxy setup on Shibuya');
-    console.log(`\na) Add a proxy for Alice on Shibuya If there is no proxy of Turing (paraId:${TuringDev.paraId}) \n`);
-    const proxyType = 'Any';
-    // const proxyType = 'DappsStaking';
-    const proxyOnShibuya = shibuyaHelper.getProxyAccount(TuringDev.paraId, shibuyaAddress);
+    console.log(`\na) Add a proxy for Alice If there is none setup on Shibuya (paraId:${TuringDev.paraId}) \n`);
+
+    const proxyTypeShibuya = 'Any'; // We cannotset proxyType to "DappsStaking" without the actual auto-restake call
+    const proxyOnShibuya = shibuyaHelper.getProxyAccount(shibuyaAddress, TuringDev.paraId);
```
Comment (chrisli30) [inline src/shibuya.js:113]: I change the order of parameter in this getProxyAccount, because it makes more sense to have the wallet address in the front.

**Review by imstar15 — APPROVED** [review]
Look good to me!

**Pull request merged**